RealSense_Height: 0.98
Arm_Height: 0.78
Shoulder_Elbow_Length: 0.22
Elbow_Wrist_Length: 0.17
Endeffector_Length: 0.15